keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -277,6 +283,10 @@ know that this has succeeded?
 - Handling of Pod configuration errors resulting in pods stuck in the `Pending`
   state (value of `status.phase`) rather than `Failed` (such as incorrect image
   name, non-matching configMap references, incorrect PVC references).
+- Adding of `ResourceExhausted` condition for Windows containers.
```

Comment:
FYI @kubernetes/sig-windows-leads

Reply:
I read the KEP updates and just wanted to confirm that only the scenarios called out here (OOM and node shutdown issues) were **not** expected to work for Windows workloads correct?
If that is the case this I don't see any issues with these updates with respect to Windows support.

Reply:
Correct. Thanks for reviewing the changes.